src/api/spec/models/bs_request_spec.rb

```diff
@@ -88,4 +88,104 @@
       end
     end
   end
+
+  describe '#update_cache' do
+    RSpec.shared_examples 'after_commit callback' do
+      it 'touches the user' do
+        Timecop.travel(1.minute)
+        cache_key = user.cache_key
+        request.state = :review
+        request.save
+        user.reload
+        expect(user.cache_key).not_to eq(cache_key)
+      end
+    end
+
+    context 'creator of bs_request' do
+      let!(:request) { create(:bs_request, creator: user.login) }
+      let(:user) { create(:admin_user) }
+
+      include_examples 'after_commit callback'
+    end
+
+    context 'direct maintainer of a target_project' do
+      let(:target_project) { create(:project) }
+      let(:source_package) { create(:package) }
+      let(:source_project) { source_package.project }
+
+      let!(:request) do
```

Comment:
Why are you using `let!` here? Is `let` not enough? Why does it need to be created before running an `it` block?